src/app/recipebook-detail/recipebook-detail.component.ts

```diff
@@ -25,7 +26,8 @@ if (confirm("Are you sure you want to delete this recipe? This request cannot be
 
   recipe: Recipe;
   user: User;
-
+  ingredients: Recipe[]
```

Comment:
Update this to `ingredients: Array<string>;` , we aren't actually using the model here.